conf/core/GrailDifferentialConflation.conf

```diff
@@ -5,7 +5,7 @@
   "element.cache.size.relation": "2000000",
   "element.cache.size.way": "2000000",
   "map.cleaner.transforms": "hoot::ReprojectToPlanarOp;hoot::DuplicateWayRemover;hoot::SuperfluousWayRemover;hoot::IntersectionSplitter;hoot::UnlikelyIntersectionRemover;hoot::DualWaySplitter;hoot::ImpliedDividedMarker;hoot::DuplicateNameRemover;hoot::SmallWayMerger;hoot::RemoveEmptyAreasVisitor;hoot::RemoveDuplicateAreaVisitor;hoot::NoInformationElementRemover;hoot::CornerSplitter;hoot::RubberSheet",
-  "match.creators": "hoot::HighwayMatchCreator;hoot::ScriptMatchCreator,Area.js;hoot::BuildingMatchCreator;hoot::ScriptMatchCreator,PoiGeneric.js;hoot::PoiPolygonMatchCreator;hoot::ScriptMatchCreator,PowerLine.js;hoot::ScriptMatchCreator,Railway.js;hoot::ScriptMatchCreator,LinearWaterway.js",
+  "match.creators": "hoot::HighwayMatchCreator;hoot::ScriptMatchCreator,Area.js;hoot::BuildingMatchCreator;hoot::ScriptMatchCreator,Poi.js;hoot::PoiPolygonMatchCreator;hoot::ScriptMatchCreator,PowerLine.js;hoot::ScriptMatchCreator,Railway.js;hoot::ScriptMatchCreator,River.js",
```

Comment:
This is out of date now, but I didn't want to mess with their testing config. Probably doesn't matter for what they're doing if it is out of date.